doc/api/net.md

```diff
@@ -1798,7 +1798,9 @@ added: v19.4.0
 
 Sets the default value of the `autoSelectFamily` option of [`socket.connect(options)`][].
 
-* `value` {boolean} The new default value. The initial default value is `false`.
+* `value` {boolean} The new default value.
+  The initial default value is `true`, unless the command line option
+  `--no-network-family-autoselection` is provided.
```

Comment:
```suggestion
* `value` {boolean} The new default value.
  **Default:** `true` when [`--no-network-family-autoselection`][] is not used, `false` otherwise.
```

Reply:
I'd prefer not to add `**Default**` keyword, because `value` is a required parameter. Maybe it is better to move my changes to method description instead of `value` parameter description? It would be more consistent with [net.getDefaultAutoSelectFamily()](https://github.com/nodejs/node/blob/main/doc/api/net.md#netgetdefaultautoselectfamily).

Reply:
I also prefer not to use the `**Default**`.